tests/test_db_utils_close_manager_coverage.py

```
@@ -0,0 +1,118 @@
+    clear_db_path_cache()
+    _reset_db_manager()
+
+
+class TestCloseManagerSafelyKeyboardInterrupt:
+    """Tests for _close_manager_safely re-raising KeyboardInterrupt."""
+
+    def test_keyboard_interrupt_is_reraised(self):
+        mock_mgr = type(
+            "FakeMgr",
+            (),
+            {"close": lambda self: (_ for _ in ()).throw(KeyboardInterrupt("ctrl-c"))},
```

Ruff (0.15.10)

[warning] Unused lambda argument: `self`

(ARG005)